### ISSUE TITLE
servo settings

### ISSUE THREAD

**Issue by mazza8098** [luetzel/motion_eye_servo_action#1]
how do you set up the min max on the servos. when i press any button the servos just jump and go back to where they were.

**Comment by luetzel** [comment]
Simply edit files up_1, down_1, left_1, right_1 and change min/max values to your desired position.
Here, I use cheap SG90 servos. You might have to use other values with other types of servos.
Also make sure that *pos files have correct permissions, so that python scripts are able to store the current position.

`# Configure min and max servo pulse lengths (450 total, 25 corresponds to 10 degrees)
servo_min = 225  # Min pulse length out of 4096
servo_max = 725 # Max pulse length out of 4096
`

**Comment by mazza8098** [comment]
Ok perfect I have it working with the same servos but it is very jumpy not smooth at all. Any ideas on how to fix it?

**Comment by luetzel** [comment]
Did you hook up your servos using a controller board?
https://learn.adafruit.com/adafruit-16-channel-servo-driver-with-raspberry-pi/hooking-it-up
I strongly recommend to use a board with capacitor.
It is also possible to control the enable pin of the board in order to disconnect all servos from power while they are not in use.

**Comment by mazza8098** [comment]
Yes I used the control board listed. I plugged in all the wires listed as well including gpio 18 which I’m guessing is the on off wire.

Nathan Mazza


On Jan 22, 2019, at 2:22 PM, luetzel <notifications@github.com<mailto:notifications@github.com>> wrote:


Did you hook up your servos using a controller board?
https://learn.adafruit.com/adafruit-16-channel-servo-driver-with-raspberry-pi/hooking-it-up<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.adafruit.com%2Fadafruit-16-channel-servo-driver-with-raspberry-pi%2Fhooking-it-up&data=02%7C01%7C%7C73f7e5b92b9f4122c0b708d680b80f54%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636837925346382629&sdata=gMxBWlU%2FifxqOcJzudaHqoD4bvWVJnM1TBtc1HduKKo%3D&reserved=0>
I strongly recommend to use a board with capacitor.
It is also possible to control the enable pin of the board in order to disconnect all servos from power while they are not in use.
